docs/reference/overview_of_strategies.rst

```diff
@@ -141,29 +141,29 @@ repository.
    "K68R_", "Fransois Leyvraz", "Not Implemented"
    "K69R_", "Johann Joss", "Not Implemented"
    "K70R_", "Robert Pebly", "Not Implemented"
-   "K71R_", "James E Hill", "Not Implemented"
+   "K71R_", "James E Hall", "Not Implemented"
    "K72R_", "Edward C White Jr", "Not Implemented"
-   "K73R_", "Geroge Zimmerman", "Not Implemented"
+   "K73R_", "George Zimmerman", "Not Implemented"
    "K74R_", "Edward Friedland", "Not Implemented"
    "K74RXX_", "Edward Friedland", "Not Implemented"
-   "K75R_", "P D Harrington", "Not Implemented"
+   "K75R_", "Paul D Harrington", "Not Implemented"
    "K76R_", "David Gladstein", "Not Implemented"
    "K77R_", "Scott Feld", "Not Implemented"
    "K78R_", "Fred Mauk", "Not Implemented"
    "K79R_", "Dennis Ambuehl and Kevin Hickey", Not Implemented
-   "K80R_", "Robyn M Dawes and Mark Batel", Not Implemented
+   "K80R_", "Robyn M Dawes and Mark Batell", Not Implemented
    "K81R_", "Martyn Jones", "Not Implemented"
    "K82R_", "Robert A Leyland", "Not Implemented"
    "K83R_", "Paul E Black", "Not Implemented"
    "K84R_", "T Nicolaus Tideman and Paula Chieruzz", "Not Implemented"
-   "K85R_", "Rober B Falk and James M Langsted", "Not Implemented"
-   "K86R_", "Bernard Grofman", "Not Implemented"
+   "K85R_", "Robert B Falk and James M Langsted", "Not Implemented"
+   "K86R_", "Bernard Grofman", "`Issue 1093 <https://github.com/Axelrod-Python/Axelrod/issues/1093>`_"
```

Comment:
I did this because otherwise we have no other reference point for the question "Which of the second tournament strategies do we still need to look at and reverse engineer"